admin/class-meta-columns.php

```diff
@@ -5,11 +5,22 @@
  * @package WPSEO\Admin
  */
 
+use Yoast\WP\SEO\Context\Meta_Tags_Context;
+use Yoast\WP\SEO\Memoizers\Meta_Tags_Context_Memoizer;
+use Yoast\WP\SEO\Repositories\Indexable_Repository;
+
 /**
  * Class WPSEO_Meta_Columns.
  */
 class WPSEO_Meta_Columns {
 
+	/**
+	 * Holds the context objects for each indexable.
+	 *
+	 * @var Meta_Tags_Context[]
+	 */
+	protected $indexables;
```

Comment:
I'm a bit confused about the name here. This is the context that is also containing the indexable. Why call it `$indexables` and not `$context` or something?